ember/app/abilities/work-item.js

```diff
@@ -1,13 +1,31 @@
+import { inject as service } from "@ember/service";
+
 import BaseAbility from "mysagw/abilities/-base";
+import ENV from "mysagw/config/environment";
 
 export default class WorkItemAbility extends BaseAbility {
+  @service store;
+
   get canList() {
-    return this.isStaff;
+    const identity = this.store
+      .peekAll("identity")
+      .findBy("idpId", this.session.data.authenticated.userinfo?.sub);
+
+    const nwpIdentity = identity?.memberships.find(
+      (membership) =>
+        ENV.APP.circulationOrganisations.includes(
+          membership.organisation.get("organisationName")
+        ) && !membership.isInactive
+    );
```

Comment:
Suspect for the custom `session` service. :man_shrugging:

Reply:
Moved aswell